parsigs/parse_sig_api.py

```diff
@@ -87,26 +88,39 @@ def _parse_sig(sig: str, model: Language):
     return _create_structured_sigs(model_output)
 
 
-def _pre_process(sig):
-    sig = sig.lower().replace('twice', '2 times').replace("once", '1 time').replace("nightly", "every night")
+def _autocorrect(sig):
+    spell = SpellChecker()
+    spell.word_frequency.load_text_file(os.path.join('data', 'drug_names.txt'))
+    sig = sig.lower().strip()
+    corrected_words = []
+    for word in sig.split():
+        if not re.match(r"^[a-zA-Z]+$",word) or spell.known([word]):
```

Comment:
define the REGEX as a global constant with a meaningful name and use it here

Reply:
After a conversation on the project's Discord, I'll write a short record for this regex.